src/slider.jsx

```diff
@@ -323,17 +314,84 @@ const Slider = React.createClass({
       document.addEventListener('touchcancel', this._dragTouchEndHandler, false);
     }
     this._onDragStart(e);
+
+    // Cancel scroll and context menu
+    e.preventDefault();
   },
 
   _onHandleMouseDown(e) {
     if (document) {
       document.addEventListener('mousemove', this._dragHandler, false);
       document.addEventListener('mouseup', this._dragEndHandler, false);
-      this._toggleSelection('none');
+
+      // Cancel text selection
+      e.preventDefault();
+
+      // Set focus manually since we called preventDefault()
+      ReactDOM.findDOMNode(this.refs.handle).focus();
     }
     this._onDragStart(e);
   },
 
+  _onHandleKeyDown(e) {
+    const pageDown = 34;
```

Comment:
We would need to expand a little the key-code.js, but I think it is a nice idea use it, because the key codes are constants that a lot of components may want to use, so it makes sense to me have them as a separate module. Thanks for the tip!